.run/Run - Platform Server - no Docker.run.xml

```diff
@@ -0,0 +1,10 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Run - Platform Server - no Docker" type="JetRunConfigurationType">
```

Comment:
@lijamie98, as you updated the other run configurations from `Docker` to `fullstack`, shouldn't you do the same here?